sys/net/application_layer/coap/gcoap.c

```diff
@@ -452,6 +452,7 @@ static ssize_t _write_options(coap_pkt_t *pdu, uint8_t *buf, size_t len)
         size_t url_len = strlen((char *)pdu->url);
         if (url_len) {
             if (pdu->url[0] != '/') {
```

Comment:
hä, this is redundant, because already done in `_write_options`, though I oppose that, too (see above).

mhm, maybe leave as is and fix only the methods code and docu stuff. And fix this url checking in a separate PR?